File: client/java/src/avt/ImageIcon.java
package avt;

import javax.microedition.lcdui.Image;

public final class ImageIcon {
   public Image img;
   public short w;
   public short h;
   public int count = -1;

   public ImageIcon() {
   }

   public ImageIcon(Image var1) {
      this.img = var1;
      this.count = 0;
      this.w = (short)var1.getWidth();
      this.h = (short)var1.getHeight();
   }
}

File: client/unity/Assets/Scripts/Assembly-CSharp/ImageIcon.cs
public class ImageIcon
{
	public Image img;

	public short w;

	public short h;

	public int count = -1;

	public ImageIcon()
	{
	}

	public ImageIcon(Image im)
	{
		img = im;
		count = 0;
		w = (short)im.getWidth();
		h = (short)im.getHeight();
	}
}
